index.html

```diff
@@ -550,14 +550,15 @@ <h4>Supported Python</h4>
                   </p>
                 </div>
               </div>
+
```

Comment:
```suggestion
```

Reply:
I prefer to have these extra newlines to make it easier to see where the tags start and end